### DISCUSSION TITLE
Sites deployed to GitHub Pages and Gitee Pages do not work

### DISCUSSION THREAD

**Discussion by w1135254654** [vuejs/vitepress#3094] [Q&A]
On GitHub Pages

According to the configuration requirements in the document,base: '/ljvitepress/' has been added to /.vitepress/config.js
![config](https://github.com/vuejs/vitepress/assets/30262526/5e875c65-6746-4171-af56-d0088fb5df9f)
GitHub Pages are configured
![github](https://github.com/vuejs/vitepress/assets/30262526/ebc09d66-0e46-4e51-977a-21a320bb6eb9)
The Edge browser does not take effect after being accessed, why! why! why!
![Edge](https://github.com/vuejs/vitepress/assets/30262526/98b6fbb5-3d7d-40bd-8763-d03a9e99840a)

On Gitee Pages

The deployment process is the same,base: '/ljvitepress/' has been added to /.vitepress/config.js

Gitee Pages are configured
![gitee](https://github.com/vuejs/vitepress/assets/30262526/dbdc4b2e-cbd8-4ad6-9c67-33c8596fcd00)
Error 404 in Edge browser, why! why! why!
![404](https://github.com/vuejs/vitepress/assets/30262526/d9887233-aaf2-42f2-9aa5-09474212dc50)

Thanks for helping with that🙏

**Comment by brc-dd** [comment]
Refer this for github pages -- https://vitepress.dev/guide/deploy#github-pages

**Comment by sharpchen** [comment]
Check deployment in the workflow, it probably failed. And solve those problems by errors from container

**Comment by elarbor** [comment]
同问
我要部署gitee，github有点卡
